File: team/awnser.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func isProblemImplemented(petyaConfidence, vasyaConfidence, tonyaConfidence int) bool {
	sumOfConfidences := petyaConfidence + vasyaConfidence + tonyaConfidence
	return sumOfConfidences >= 2
}

func main() {
	inputReader := bufio.NewReader(os.Stdin)
	outputWriter := bufio.NewWriter(os.Stdout)
	defer outputWriter.Flush()

	var problemsCount int
	if _, err := fmt.Fscan(inputReader, &problemsCount); err != nil {
		return
	}

	implementedProblemsCount := 0

	for problemIndex := 0; problemIndex < problemsCount; problemIndex++ {
		var petyaConfidence, vasyaConfidence, tonyaConfidence int
		fmt.Fscan(inputReader, &petyaConfidence, &vasyaConfidence, &tonyaConfidence)

		if isProblemImplemented(petyaConfidence, vasyaConfidence, tonyaConfidence) {
			implementedProblemsCount++
		}
	}

	fmt.Fprintln(outputWriter, implementedProblemsCount)
}

File: team/awnser.py
import sys
from typing import Iterable


def is_problem_implemented(confidences: Iterable[int]) -> bool:
    return sum(confidences) >= 2


def main() -> None:
    input_data = sys.stdin.read().strip().split()
    if not input_data:
        return

    problems_count = int(input_data[0])
    index = 1

    implemented_problems_count = 0

    for _ in range(problems_count):
        petya_confidence = int(input_data[index])
        vasya_confidence = int(input_data[index + 1])
        tonya_confidence = int(input_data[index + 2])
        index += 3

        if is_problem_implemented(
            (petya_confidence, vasya_confidence, tonya_confidence)
        ):
            implemented_problems_count += 1

    print(implemented_problems_count)


if __name__ == "__main__":
    main()
